src/layouts/components/Editable/Editable.tsx

```diff
@@ -0,0 +1,241 @@
+          clicking “Add section” below
+        </Text>
+      </Flex>
+    </VStack>
+  )
+}
+
+export interface EditableSidebarProps {
+  title: string
+}
+const EditableSidebar = ({
+  title,
+  children,
+}: PropsWithChildren<EditableSidebarProps>) => {
+  return (
+    <Flex flexDir="column" bg="base.canvas.alt">
+      <SidebarHeader title={title} />
+      {children ? <>{children}</> : <EmptySideBarBody />}
+    </Flex>
+  )
+}
+
+type HomepageDroppableZones = "dropdownelem" | "leftPane" | "highlight"
+
+type DropInfo = {
+  droppableId: HomepageDroppableZones
+  type: string
+}
+
+const getDroppableInfo = (editableId: HomepageDroppableZones): DropInfo => {
+  if (editableId === "leftPane")
+    return { droppableId: editableId, type: "editor" }
+  return {
+    droppableId: editableId,
+    type: editableId,
+  }
+}
+
+export interface EditableDraggableProps {
+  onDragEnd: OnDragEndResponder
+  editableId: HomepageDroppableZones
+}
+const EditableDraggable = ({
+  onDragEnd,
+  children,
+  editableId,
+}: PropsWithChildren<EditableDraggableProps>) => {
+  return (
+    // NOTE: According to the dnd docs,
+    // there CANNOT be more than 1
+    // `DragDropContext` in the component tree.
+    <DragDropContext onDragEnd={onDragEnd}>
+      <Droppable {...getDroppableInfo(editableId)}>
+        {(droppableProvided) => (
+          <div
+            ref={droppableProvided.innerRef}
+            {...droppableProvided.droppableProps}
+          >
+            {children}
+            {droppableProvided.placeholder}
+          </div>
+        )}
+      </Droppable>
+    </DragDropContext>
+  )
+}
+
+const BaseAccordionItem = forwardRef((props: AccordionItemProps, ref) => {
+  return (
+    <AccordionItem
+      border="1px solid"
+      borderColor="base.divider.medium"
+      w="100%"
+      bg="base.canvas.default"
+      ref={ref}
+      {...props}
+    />
+  )
+})
+
+interface EditableCardProps {
+  title: string
+}
+// TODO: Break this up into individual sub-components
+// that can be selectively used to create the card
+const EditableAccordionItem = ({
+  title,
+  children,
+}: PropsWithChildren<EditableCardProps>) => {
+  return (
+    <BaseAccordionItem borderRadius="0.5rem">
+      {/* NOTE: Check with design on styling. 
+        See if entire section is button (ie, whole component hover styling)
+      */}
+      <AccordionButton px="1.5rem" py="3rem">
+        <Flex flex="1" flexDir="column">
+          <Text textStyle="h6" textAlign="left" mt="0.25rem">
+            {title}
+          </Text>
+        </Flex>
+        <AccordionIcon />
+      </AccordionButton>
+      <AccordionPanel pb={4}>{children}</AccordionPanel>
+    </BaseAccordionItem>
+  )
+}
+
+interface DraggableAccordionItemProps {
+  tag: JSX.Element
+  title: string
+  // TODO: Should get these props automatically
+  // rather than having us pass in manually
+  index: number
+  draggableId: string
+}
+// NOTE: Separating editable/draggable
```

Comment:
may I clarify that all editable is a dragable?

Reply:
no - this is for accordion items only. editable accordion item (only 1 eg -  the homepage) don't have dragging capabilities + diff styling. draggable accordion items can be dragged.